src/main/java/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.FileReader;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> listOfLines = new ArrayList<>();
+        File file = new File(fileName);
```

Comment:
Avoid using the constructor `new File()` directly in the service implementation. It's better to pass the `fileName` to the `BufferedReader` directly without creating a `File` object. Also, ensure that the `fileName` is a relative path to the resources folder.